content/content-strategy/define-user-content-needs/create-actionable-artefacts/user-profiles/new-long-description.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: blocks/long_description
+---
+### [1]User profile: long text description 
```

Comment:
Incorrect display level. Needs to be `### [2]` to show a heading 3.

Reply:
Fixed

Reply:
Steve fixed